package.json

```diff
@@ -47,20 +47,19 @@
     "NOTICE"
   ],
   "scripts": {
+    "start": "vite",
```

Comment:
Without read the description of the PR, I didn't understand this command is the old `watch` command.
Maybe need to rename it.

In addition, I don't know if we can do that, but it would be better if the command start directly the demo page, and the user doesn't have to know the url http://localhost:10001/dev/public/index.html to go to the demo.

Reply:
1. for the script name, we can rename it into `dev`. This is what is proposed in vitejs templates
2. for the start url, we can at least create a index.html that redirects to dev/public/index.html - see the `To continue if we implement vitejs for real` paragraph in the PR description

Reply:
available now
1. done in d453aa51
2. done in 4c7ab8ea